demo/api/src/ml.module.ts

```diff
@@ -0,0 +1,84 @@
+import { MlRequest, Options } from "@comet/cms-api/lib/ml/ml.types";
+
+export const MLModels = () => {
```

Comment:
No need for this to be a function, you can simply export the object, right?

Reply:
Why would i want to make it into an object?

Reply:
Currently, your function looks like this:

```ts
export const MLModels = () => {
  return {} // an object
}
```

As you're not using an arguments in the function, the function isn't necessary. So you can simply do this:

```ts
export const mlModes = {
// Same things as in the returned object
}
```

Reply:
I now parse the API URL and API key into the function. Should I still convert it?